src/core/public/plugins/plugin_context.ts

```diff
@@ -100,7 +102,7 @@ export function createPluginStartContext<
     },
     docLinks: deps.docLinks,
     http: deps.http,
-    chrome: deps.chrome,
+    chrome: omit(deps.chrome, 'getComponent'),
```

Comment:
Can you elaborate on why this needs to happen?

Reply:
This function is internal to Core (used by the RenderingService). Though this function is not exposed on the ChromeStart type I still want to make sure it's not callable at all from plugins.